awpy/types.py

```diff
@@ -548,6 +548,9 @@ class GameRound(TypedDict):
     winningTeam: str | None
     losingTeam: str | None
     roundEndReason: str
+    MVPName: str | None
```

Comment:
for naming convention purposes, we can make these `mvpName`, `mvpSteamId`, and so on